### PR TITLE
Add Coverage Exclude tester

### LINKED ISSUE #25: Add coverage exclude tester
Add a tester to inspect a project's config and look for coverage excludes. 0 is the optimal number here.

### PR DESCRIPTION
**Pull request by baer**
Like #32 this is based off of a develop branch à la Git Flow. See #32 for explanation if you get to this first.

This PR closes #25

### CODE REVIEW

File: lib/tests/coverageExcludes.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+var baseConfig = {};
+
+var ComplexityTest = new krabbyTest(baseConfig);
+
+ComplexityTest.prototype.getConfig = function(filePath) {
+  var fileExists = fs.lstatSync(filePath).isFile();
+  return fileExists ? require(filePath) : null;
+}
+
+// Uses a binomial to decrease the score at a near exponential rate with full failure at 13 excludes
```
Comment (patrickkettner) [inline lib/tests/coverageExcludes.js:17]: `13` should be configurable, right?

**Comment by patrickkettner** [comment]
lgtm - seems like that the limit should be configurable, however

**Comment by baer** [comment]
I completely agree good call. feel free to take this branch and add that
commit or merge it and do it afterwards. Don't forget this PR is against to
develop if you'd like to merge to master now the 30s in make sure you
redirect it

**Comment by patrickkettner** [comment]
sounds good - merging into master, will update shortly with another PR

**Comment by patrickkettner** [comment]
erp - can't change the branch destination - are you able to? otherwise I will clone yours and merge locally

**Comment by baer** [comment]
I can't change it either... I will merge it from the cmd line and push

**Comment by patrickkettner** [comment]
thanks mang :+1:

**Comment by baer** [comment]
Closing and merging to master instead of develop via CLI

**Pull request closed without merging**